timelock-agent/src/main/java/com/palantir/timelock/config/PaxosTsBoundPersisterConfiguration.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.timelock.config;
+
+import org.immutables.value.Value;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+@JsonDeserialize(as = ImmutablePaxosTsBoundPersisterConfiguration.class)
+@JsonSerialize(as = ImmutablePaxosTsBoundPersisterConfiguration.class)
+@Value.Immutable
+public abstract class PaxosTsBoundPersisterConfiguration implements TsBoundPersisterConfiguration {
+
+    @Value.Default
+    public PaxosInstallConfiguration paxos() {
```

Comment:
Uncomfortable with a default here. I think we should make this value optional, and if it's not specified than we fall back to the already required configuration value stored at `install.paxos`.

Reply:
Make the paxos config required in `PaxosTsBoundPersisterConfiguration` and the Install config will default to a `PaxosTsBoundPersisterConfiguration` with the paxos config specified at the top-level.